.github/workflows/test-e2e.yml

```
@@ -62,11 +62,11 @@ jobs:
     needs:
       - build-aggkit-image
       - read-aggkit-args
-    uses: agglayer/e2e/.github/workflows/aggkit-e2e-single-chain.yml@60fdb6e88e57083bd21daae01483df048245f89e
+    uses: agglayer/e2e/.github/workflows/aggkit-e2e-single-chain.yml@f32a8e6bdce82f22c2bcc6991993d728f39d3bb4
     secrets: inherit
     with:
-      kurtosis-cdk-ref: 45899b00758ae2e8b4f1504c1d711a818fd652f7
-      agglayer-e2e-ref: 60fdb6e88e57083bd21daae01483df048245f89e
+      kurtosis-cdk-ref: 665e86e051e3f2ddcdd2e1dfb05faceb1b2edb5d
```

Should we use this ref for kurtosis?

```suggestion
      kurtosis-cdk-ref: 9fc300b72b0ff28b071f6851c9718d654a1dfdde
```